support/src/setupProxy.js

```diff
@@ -179,18 +155,18 @@ const mockNoResultsAPI = (err) => {
  * @param {Express.Application} app
  */
 const middleware = (app) => {
-	app.use(
-		'/api/Terms/:queryType/:dictionary/:audience/:language/:query',
-		getDictionarySearchResults
-	);
-
 	app.use(
 		'/api/sitewidesearch/v1/Search/:collection/:language/:term',
 		getSearchResults
 	);
 
 	app.use('/api/BestBets/:collection/:language/:term', getBestBetsResults);
 
+	app.use(
```

Comment:
Why hardcode what dictionary this will use?  Remember that this library could be used by other people for other purposes and applications.  I would switch back to the configurable path

Reply:
👍

Reply:
Done!